tests/wasm/generate_tests.sh

```diff
@@ -40,4 +40,6 @@ chmod +x gen.sh
 cat modules.txt | xargs -n1 -P"$cores" ./gen.sh
 rm gen.sh
 
+mv test_callbacks.skip test_callbacks.py
```

Comment:
Why is this needed?

Reply:
The WASM examples don't exhibit all the behaviors that Manticore can fire callbacks for, so `test_callbacks.skip` draws on examples from the spec tests for things like global variable usage, traps, and memory usage. Since those tests won't exist unless you've run generate_tests.sh, I disabled `test_callbacks` until you've done so by renaming the file.

Reply:
Fair enough! My main concern was that the `test_callbacks.skip` might not be formatted or linted (if we ever eventually lint tests) by tools that look for the `*.py` extension.